frontend/packages/console-shared/src/components/dashboard/resource-quota-card/AppliedClusterResourceQuotaItem.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Text, TextVariants } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import {
+  getQuotaResourceTypes,
+  QuotaScopesInline,
+  QuotaGaugeCharts,
+} from '@console/internal/components/resource-quota';
+import { ResourceLink } from '@console/internal/components/utils/resource-link';
+import { AppliedClusterResourceQuotaModel } from '@console/internal/models';
+import { referenceForModel, AppliedClusterResourceQuotaKind } from '@console/internal/module/k8s';
+
+import './resource-quota-card.scss';
+
+const AppliedClusterResourceQuotaItem: React.FC<AppliedClusterResourceQuotaItemProps> = ({
+  resourceQuota,
+  namespace,
+}) => {
+  const resourceTypes = getQuotaResourceTypes(resourceQuota);
+  const scopes = resourceQuota?.spec?.scopes;
+  const { t } = useTranslation();
+  return (
+    <>
+      <div>
+        <ResourceLink
+          kind={referenceForModel(AppliedClusterResourceQuotaModel)}
+          name={resourceQuota.metadata.name}
+          className="co-resource-item--truncate co-resource-quota-card__item-title"
+          namespace={namespace}
+          inline
+        />
+        {scopes && <QuotaScopesInline scopes={scopes} />}
+      </div>
+      <Text component={TextVariants.h4}>{t('public~For project')}</Text>
```

Comment:
It would be better to do this in one donut chart with multiple different colored arcs (usage for current project and usage for other projects).

Reply:
I wrote to Chris to see if we can get a design done for this.

Reply:
Do you mean something like this? https://www.sketch.com/s/a2668252-07fe-4472-a96f-d3bf94423959/a/jgpErYb (Sorry, some of the formatting has been messed up)

Reply:
I'd definitely like to hear what @cshinn thinks, but this could probably be a simple donut chart with an arc for the current project and an arc for other projects. That's what we did in OpenShift 3.x.

Reply:
It looks like PatternFly won't let you do gauge charts like that design @cshinn. ChartDonutUtilization only lets you pass in one x and y coordinate. It looks like we can do a simple donut chart like @spadgett is suggesting (it looks like those let you pass in more data), but you can potentially wind up with charts that look like this (where both the cluster and namespace data are at 25%):
<img width="239" alt="Screen Shot 2021-11-05 at 8 06 05 PM" src="https://user-images.githubusercontent.com/7014965/140590824-782a534f-fa9b-4353-9ab3-bf0b76d6b004.png">

Are you cool with this or do you want to do something else?

Reply:
hmm, I think that could be misleading. It's kinda kludgy, but would it be possible to make a "fake" segment that represented the unused portion?

Reply:
if that's not possible, I guess it's probably best to stick with the separate charts for the project and total usage and make a feature request against PatternFly about being able to have multiple series in a utilization chart

Reply:
@spadgett - I spoke to @cshinn on Slack and he said it seems like the best case for now would be to use separate charts like we had before, but maybe it’s worth making a feature request against PF. Are you ok with that?